.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 exclude =
     six.py
+ignore = E501, W503
```

Comment:
`E501` is the max line length error. Black makes a best effort to respect the maximum line length, but sometimes goes over if other rules require it. Black's own [documentation](https://github.com/ambv/black#line-length) recommends ignoring `E501`.

`W503` is a non-PEP8 compliant warning that is ignored by default, but must be explicitly ignored when an explicit `ignore` configuration line is provided.

Reply:
Could we add a comment over this with the content of your description here (these alphanumerical designations are pretty obtuse).

Reply:
👍 Done.